components/FeaturesConfigDialog.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  PrimaryButton,
+  Stack,
+  Dialog,
+  IStackTokens,
+  Checkbox,
+} from "@fluentui/react";
+import * as React from "react";
+import {
+  defaultFeaturesConfig,
+  featuresCache,
+  setFeaturesConfig,
+} from "../utils/featuresConfigUtils";
+import type {
+  FeaturesConfig,
+  Component,
+  Feature,
+} from "../utils/featuresConfigUtils";
+
+type Props = {
+  accountId: string;
+  onDismiss: () => void;
+  onSave: (featuresConfig: FeaturesConfig) => void;
+};
+
+function deepCopy(obj: Object) {
+  return JSON.parse(JSON.stringify(obj));
+}
+
+export const FeaturesConfigDialog: React.FC<Props> = (props) => {
+  const renderContent = () => {
+    const tokens: IStackTokens = {
+      childrenGap: "15px",
+    };
+
+    const [featuresConfigState, setFeaturesConfigState] = React.useState(
+      deepCopy(
+        featuresCache[props.accountId]?.featuresConfig || defaultFeaturesConfig,
+      ),
+    );
+
+    return (
+      <Stack tokens={tokens}>
```

Comment:
Generally stacks contain stack items, so Stack -> StackItem. Not This isn't a strong point here since this is test/debug UI but I think some of these stacks should be StackItems!